progress-bar.dust

```diff
@@ -0,0 +1,24 @@
+<div class="c-progress-bar c--{state} {?isRadial}c--radial{/isRadial} {class}" {?id}id="{.}"{/id} role="progressbar" aria-valuenow="{value}" aria-valuemin="0" aria-valuemax="1" aria-valuetext="{label}" data-adaptivejs-component="stencil-progress-bar">
+
+    {?isRadial}
+        {@c-progress-bar__radial/}
+    {/isRadial}
+
+    <div class="c-progress-bar__middle">
```

Comment:
I wonder if `middle` might be a confusing sub-class name? It's not always the middle, and the other elements at this nesting level aren't following any kind of ordering naming convention. Visually, it always looks like it's on the left or right.